### PR TITLE
feat: title, axis theme

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

#### title

```ts
type FontTheme = {
  fontWeight: number | string;
  color: string;
  fontFamily: string;
  fontSize: number;
}


interface theme {
  title: FontTheme;
}
```

image

#### axis

```ts
interface AxisTheme {
  title?: FontTheme;
  label?: FontTheme;
  width?: number;
  color?: string;
}
```

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/helpers/axes.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { Options, Series, ChartOptionsUsingYAxis } from '@t/store/store';
 import { LineTypeXAxisOptions, BulletChartOptions } from '@t/options';
+import { FontTheme, Theme } from '@t/theme';
```
Comment (lja1018) [inline src/helpers/axes.ts:3]: `FontTheme`는 사용하지 않아서 제거해도 될 것 같습니다~

File: src/store/theme.ts [hunk 2]
```diff
@@ -33,19 +34,46 @@ function getCommonSeriesOptions(
...
+  }
+
+  const yAxis = (theme.yAxis as AxisTheme[]).map((yAxisTheme) =>
+    deepMergedCopy(
+      {
+        title: {
+          fontSize: 11,
+          fontFamily: 'Arial',
+          fontWeight: 700,
+          color: '#bbbbbb',
```
Comment (jungeun-cho) [inline src/store/theme.ts:51]: 기본 Axis Title 상수로 정의해서 사용할까요. 세번은 반복되는 것 같아서요~!

**Review by lja1018 — APPROVED** [review]
[11/10] 리뷰완료
고생하셨습니다!

**Review by jungeun-cho — APPROVED** [review]
[10/11] 리뷰완료입니다~! 폰트만 바꾸어도 분위기가 확 달라지네요!

**Comment by shiren** [comment]
LGTM

**Pull request merged**